### PR TITLE
Make workers optional for now, add a warning if they are offline

### LINKED ISSUE #489: Add warning message if workers are down
**Is your feature request related to a problem? Please describe.**
We want to move a lot of workload over to workers, but if workers are down we would silently fail to do a lot of crucial functions

**Describe the solution you'd like**
A big red warning message somewhere (maybe in the top right in the dropdown menu?) that explains workers are down

See #455 for more context

### PR DESCRIPTION
**Pull request by mariusandra**
Fixes #489, more info in that issue.

### CODE REVIEW

**Comment by mariusandra** [comment]
FYI.

The latest commit adds [redbeat](https://github.com/sibson/redbeat) as the default scheduler. This is to assure that on environments like Heroku only one heartbeat service will run at the same time. The normal scheduler uses a file in the filesystem, which will cause problems in dynamic cloud environments (heroku) and will not prevent for example two instances of the same scheduled task being run.

Redbeat is made by [the Heroku team](https://blog.heroku.com/redbeat-celery-beat-scheduler).

File: bin/start-worker [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 set -e
 
-celery -A posthog worker
+# start celery worker with heartbeat (-B)
+celery -A posthog worker -B
```
Comment (timgl) [inline bin/start-worker:5]: Does this run redbeat too?

Comment (mariusandra) [reply bin/start-worker]: Nope, this is without. I thought about adding it here, but for local development with one worker there's no need for it. Then again, it's probably best to have as much parity with production as possible... so, [added](https://github.com/PostHog/posthog/pull/494/commits/5c507ee33757ccc9d9fd198ccdf517c3ab4b2088).

**Review by timgl — APPROVED** [review]
Happy to merge!

**Pull request merged**